tests/test_communication_article.py

```diff
@@ -270,16 +294,72 @@ def test_ANSWER_DONE_to_PREPARING_ANSWER(self):
     #                          Ordering & Filtering                           #
     # ======================================================================= #
     
-    # 좋아요 개수로 정렬 확인
-    def test_ordering_by_positive_vote_count(self):
-        # TODO: In different branch
-        pass
+    # 좋아요 개수 내림차순 정렬 확인
+    def test_descending_ordering_by_positive_vote_count(self):
+        vote_counts = (0, 1, 2, 2, 3, 4)
+        articles = [self._create_article_with_status() for _ in vote_counts]
+
+        for vote_cnt, article in zip(vote_counts, articles):
+            self._add_positive_votes(article, vote_cnt)
+            article.refresh_from_db()
+        
+        res = self.http_request(self.user, 'get', 'articles',
+            querystring='ordering=-positive_vote_count,-created_at')
+        assert res.status_code == HTTP_200_OK
 
+        res_result = res.data.get('results')
+
+        # 좋아요 개수 내림차순 정렬 확인
+        res_positive_votes = [el.get('positive_vote_count') for el in res_result]
+        assert res_positive_votes == sorted(res_positive_votes, reverse=True)
+
+        # 좋아요 개수 같은 경우 최신 글이 앞에 있는지 확인
+        res_vote_cnt_eq = [el.get('created_at') for el in res_result if el.get('positive_vote_count') == 2]
+        assert res_vote_cnt_eq == sorted(res_vote_cnt_eq, reverse=True, key=lambda date_str: datetime.fromisoformat(date_str))
+    
+    # 좋아요 개수 오름차순 정렬 확인
+    def test_ascending_ordering_by_positive_vote_count(self):
+        vote_counts = (4, 3, 2, 2, 1, 0)
+        articles = [self._create_article_with_status() for _ in vote_counts]
+
+        for vote_cnt, article in zip(vote_counts, articles):
+            self._add_positive_votes(article, vote_cnt)
+            article.refresh_from_db()
+        
+        res = self.http_request(self.user, 'get', 'articles',
+            querystring='ordering=positive_vote_count,-created_at')
+        assert res.status_code == HTTP_200_OK
+
+        res_result = res.data.get('results')
```

Comment:
res_result가 제대로 오는지 개수로 확인해주세요.

```suggestion
        res_result = res.data.get('results')
        assert len(Article.objects.all()) == len(res_result)
```